utils/config/daos_server.yml

```diff
@@ -329,7 +329,7 @@
 #
 #  # Force specific path for DAOS debug logs.
 #
-#  # default: /tmp/daos.log
+#  # default: engine log goes to control_log_file
```

Comment:
Huh, interesting. I haven't tried to run DAOS without defining an engine log in recent days, didn't realize we were spitting it out to the control log.